src/js/components/Booking.js

```diff
@@ -194,8 +187,86 @@ class Booking {
     });
 
     thisBooking.dom.wrapper.addEventListener('updated', function(){
+      for (let table of thisBooking.dom.tables) {
+        table.classList.remove(classNames.booking.tableSelected);
+      }
       thisBooking.updateDOM();
     });
+
+    thisBooking.dom.tablesContainer.addEventListener('click', function(event){
+      thisBooking.initTables(event);
+    });
+
+    thisBooking.dom.submit.addEventListener('submit', function(event) {
+      event.preventDefault();
+      thisBooking.sendBooking();
+    });
+  }
+
+  initTables(event){
+    const thisBooking = this;
+
+    const clickedElem = event.target;
+    event.preventDefault();
+
+    const tableId = clickedElem.getAttribute(settings.booking.tableIdAttribute);
+
+    if(tableId) {
+      if(clickedElem.classList.contains(classNames.booking.tableBooked)){
+        alert('Table unavailable');
+      } else {
+        if (clickedElem.classList.contains(classNames.booking.tableSelected)){
+          clickedElem.classList.remove(classNames.booking.tableSelected);
+        } else {
+          for (let table of thisBooking.dom.tables){
+            table.classList.remove(classNames.booking.tableSelected);
+          }
+          clickedElem.classList.add(classNames.booking.tableSelected);
+          thisBooking.tableSelected = tableId;
+        }
+      }
+    }
+  }
+
+  sendBooking(){
+    const thisBooking = this;
+
+    const url = settings.db.url + '/' + settings.db.bookings;
```

Comment:
Użycie string interpolation byłoby bardziej czytelne
```suggestion
    const url = `${settings.db.url}/${settings.db.bookings}`;
```